src/Types.ts

```diff
@@ -164,8 +165,8 @@ export interface ConditionArgsBase {
 
 export interface PriceConditionArgs extends ConditionArgsBase {
   operator: PriceOperator
-  tokenA: string
```

Comment:
@phanpak I think these should actually be `tokenA: string | TokenArgs` and the user can provide either string for address, or full TokenArgs object

Reply:
I did this for market swap action, but I guess I forgot to do it for price condition:
https://github.com/brinktrade/brink-types/blob/30df9d78c861a4126a64bed5afb092f63fbdde9c/src/Types.ts#L196

Reply:
btw the TokenArgs type with additional params is really for future NFT support, I think we'll be encouraging devs to set address here in the near future. But good to allow TokenArgs now I think